shared_model/interfaces/query_responses/get_block_response.hpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_GET_BLOCK_RESPONSE_HPP
+#define IROHA_GET_BLOCK_RESPONSE_HPP
+
+#include "interfaces/base/model_primitive.hpp"
+#include "interfaces/iroha_internal/block.hpp"
+
+namespace shared_model {
+  namespace interface {
+    /**
+     * Provide response with the block
+     */
+    class GetBlockResponse : public ModelPrimitive<GetBlockResponse> {
```

Comment:
It looks identical to BlockResponse, why not reuse it?
https://github.com/hyperledger/iroha/blob/5671e2d1300b344714aa883f2dbf6f241ec7bb50/shared_model/interfaces/query_responses/block_response.hpp#L20